src/libraries/System.ServiceProcess.ServiceController/src/System.ServiceProcess.ServiceController.csproj

```diff
@@ -4,7 +4,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NoWarn>$(NoWarn);CA2249</NoWarn>
     <IsPackable>true</IsPackable>
-    <PackageDescription>Provides the System.ServiceProcess.ServiceContainer class, which allows you to connect to a running or stopped service, manipulate it, or get information about it.
+    <PackageDescription>Provides the System.ServiceProcess.ServiceController class, which allows you to connect to a running or stopped service, manipulate it, or get information about it.
```

Comment:
I would also mention "Windows service" here, instead of "running or stopped service".